packages/synapse-react-client/src/components/StandaloneQueryWrapper/StandaloneQueryWrapper.tsx

```diff
@@ -59,12 +55,11 @@ type StandaloneQueryWrapperOwnProps = {
   > &
   Pick<QueryWrapperPlotNavProps, 'cardConfiguration' | 'tableConfiguration'>
 
-export type StandaloneQueryWrapperProps = SynapseTableConfiguration &
-  SearchParams &
-  Operator &
+export type StandaloneQueryWrapperProps = QueryOrDeprecatedSearchParams &
```

Comment:
I didn't want to update QueryWrapperPlotNav and StandaloneQueryWrapper with this new type, but the new Portal Search components accepted both QueryWrapperPlotNav props and CardContainerLogic props, so if I changed one, I had to change all.